pkg/piecestore/psserver/psdb/psdb_test.go

```diff
@@ -147,7 +147,18 @@ func TestHappyPath(t *testing.T) {
 		bandwidthAllocation("signed by test", nodeIDAB, 3),
 	}
 
+	type BWUSAGE struct {
+		size    int64
+		timenow time.Time
+	}
+
+	bwtests := []BWUSAGE{
```

Comment:
This whole BWUSAGE thing looks pretty bogus.  I know you didn't write it though.

Reply:
Done